src/dtlssrtptransport.cpp

```diff
@@ -228,7 +228,8 @@ void DtlsSrtpTransport::incoming(message_ptr message) {
 				} else
 					PLOG_WARNING << "SRTP unprotect error, status=" << err
 					             << " SSRC=" << reinterpret_cast<RTP *>(message->data())->ssrc();
-				return;
+					if (err)
+						return;
```

Comment:
Good catch, could you please add brackets around the `PLOG_WARNING` line for clarity and fix the similar issue with `srtp_unprotect_rtcp()` a few lines higher?

Reply:
I guess we could add the same check in the "Range 64-95" block above

Reply:
Yes sounds good!